### DISCUSSION TITLE
Option to Change the Sound Event to Follow the Windows Default Playback Device or Default TeamTalk Playback Device

### DISCUSSION THREAD

**Discussion by lbk2907** [BearWare/TeamTalk5#1669] [Ideas]
### Description

Add the feature so that user can choose where the sound event will play on. Iether it will follow the default Windows playback device or default TeamTalk playback device. For now, "Default" will follow the default Windows playback device, "One by One" and "Overlapping" is follow the default TeamTalk playback device. Are that is possible? If not, at least "One by One" and "Overlapping" is follow the Windows default playback device.

### Application

- [X] qtTeamTalk
- [ ] TeamTalkAndroid
- [ ] iTeamTalk
- [ ] TeamTalkServer

**Comment by amirmahdifard** [comment]
@lbk2907 aparrently, as i looked at their workup for this feecher, they gave one by one and overlapping modes only in teamtalk sound device because apparently qt doesn't let them do one by one and overlapping modes, so they kepet the default mode as sound system, and the others are in teamtalk sound device: onnistly they did a good job if they fix the delays and the lags for the sounds to play.

**Comment by lbk2907** [comment]
@amirmahdifard
Thanks for the response.

**Comment by amirmahdifard** [comment]
@bear101 onnistly, i don't think that adding this is nesiserry to do: i think the current system is more fun, we have the default mode  as windows sound device, and we have the others as teamtalk sound device: i like to use teamtalk sound device  for the sound events because you can also stream your sound events more eezili, and if you mest up with your windows sound device, it doesn't destroy your teamtalk sound events: just please fix the delays for sound events playing: thank you.
